### ISSUE TITLE
Displaying a message when breaking a PB

### ISSUE THREAD

**Issue by tussosedan** [cs0x7f/cstimer#27]
Breaking a PB is a huge excitement, and it would be great if cstimer could display a message when a PB is broken in a session -- whether single or aoN. Perhaps even with a few bits of data like by how much, after how many solves, and after how much time (now that dates are available).

Thanks again for your work on this timer, it's awesome!

**Comment by cs0x7f** [comment]
A simple solution in https://github.com/cs0x7f/cstimer/commit/aceae204e5a1e20ea3f83ed4a8841a33176a9b74  and  https://github.com/cs0x7f/cstimer/commit/45b0693af9ff972b87015dfd2017c6b80a3a573a
When breaking a PB (actually session best), a message will scroll in the "csTimer" logo.

**Comment by cs0x7f** [comment]
In my opinion, popup notification is an impolite way to show some messages since it might interrupt the solving state in some sense. That's why I decide to use the logo instead of popup.
For sharing or screenshot, I'll try to find some other ways to satisfy them.

**Comment by tussosedan** [comment]
Agreed, popups aren't great. Maybe something like below the scramble?
Stats that could be interesting: how much faster are the PBs compared to the previous ones (e.g. 1.2 seconds faster), how many solves were done since the previous PB (e.g. the previous PB stood for 1204 solves), and how much time passed since the previous PB (e.g. 94 days).

**Comment by cs0x7f** [comment]
Maybe a message-history watcher with fashion ui would be a better way? And in the history list, more stats can be supplied.

**Comment by tussosedan** [comment]
Not sure what that means :) but any way to show the stats is great! And if flashing messages that disappear, maybe the cstimer logo could have some sort of notification dot or other indicator, and open the details.

**Comment by Heradar** [comment]
Is there any way to disable this? Its a nice change but the logo starts moving after every solve just because im naturally getting faster and my Ao500 is going down ever so slightly

**Comment by cs0x7f** [comment]
Add an option to disable the logohint in https://github.com/cs0x7f/cstimer/commit/d61e16e272ab12083c66a8bcb926acfa61b5347a
There are to many possible stats since there are configurable.

**Comment by tussosedan** [comment]
Hmm how about only up to ao12, then? That's rare enough.

**Comment by cs0x7f** [comment]
> Hmm how about only up to ao12, then? That's rare enough.

For the case that ao500 (or ao a large number) is always going down, would it be better to disable ao500 instead of logohint?

**Comment by tussosedan** [comment]
I think the message is only helpful for rare occurrences, where you don't want to miss a PB and celebrate it. In my opinion this is only for single/mo3/ao5/ao12.

And the additional statistics we discussed would also make more sense only for these, and not larger and more frequently broken averages.
